### PR TITLE
Feature/env specific spigot

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

**Comment by millerx** [comment]
This PR looks rad.

**Comment by jensengar** [comment]
@millerx thanks.

File: spigot-tests/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+pipeline {
+    agent none
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '30', daysToKeepStr: '30'))
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES')
+    }
+    stages {
+        stage('Test') {
+            agent { node EXECUTOR_MAP[env] }
```
Comment (millerx) [inline spigot-tests/Jenkinsfile]: Either default to dev_mesos or define all prod regions in the EXECUTOR_MAP

**Comment by andnp** [comment]
# [Codecov](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@9e17f3f`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##             master       #77   +/-   ##
==========================================
  Coverage          ?   100.00%           
==========================================
  Files             ?        10           
  Lines             ?       754           
  Branches          ?       134           
==========================================
  Hits              ?       754           
  Misses            ?         0           
  Partials          ?         0           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=footer). Last update [9e17f3f...1ef39cb](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by andnp** [comment]
# [Codecov](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@9e17f3f`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##             master       #77   +/-   ##
==========================================
  Coverage          ?   100.00%           
==========================================
  Files             ?        10           
  Lines             ?       754           
  Branches          ?       134           
==========================================
  Hits              ?       754           
  Misses            ?         0           
  Partials          ?         0           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=footer). Last update [9e17f3f...d0e1248](https://codecov.io/gh/MyPureCloud/purecloud-webrtc-sdk/pull/77?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: spigot-tests/index.js [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
 
+let envConfig = {};
+
+const defaultDevConfig = {
+  OAUTH_CLIENT_ID: 'ff22e32c-2948-4ff4-8f2c-1c379d28e84d',
+  ORG: 'TEST-valve-1ym37mj1kao',
+  USERNAME: 'agent-7-1ym37mj1kao@example.com',
+  PASSWORD: 'fV-qIe4HZtGM1yLr}^',
+  ENV_HOST: 'inindca.com'
+};
```
Comment (djhouseknecht) [inline spigot-tests/index.js]: Is it ok to have these in Github? I know it is just dca, but it still feels wrong.

**Pull request merged**